en/identity-server/7.1.0/docs/apis/organization-apis/restapis/organization-user-share.yaml

```diff
@@ -74,7 +74,7 @@ paths:
       x-codeSamples:
         - lang: Curl
           source: |
-            curl --location 'https://localhost:9443/o/api/server/v1/users/share' \
+            curl --location 'https://localhost:9443/t/carbon.super/o/api/server/v1/users/share' \
```

Comment:
is this intentional? we don't need to specify the parent in the request URL right?